src/voice_public.js

```diff
@@ -33,105 +33,116 @@ go.app = function() {
         self.states.add('state_start', function() {
             // Reset user answers when restarting the app
             self.im.user.answers = {};
+            return self.states.create("state_msg_receiver_msisdn");
```

Comment:
There should be a get_or_create_contact call here that creates the `user` and saves the `user_id`.